source/core/parameters_lagrangian.cc

```diff
@@ -68,6 +68,10 @@ namespace Parameters
                         "0.0",
                         Patterns::Double(),
                         "Particle surface energy");
+      prm.declare_entry("Hamaker constant particles",
+                        "40.e-20",
```

Comment:
Why this weird default value? not zero?

Reply:
It can be set to zero... it was the value used in the article I used to implement the model. 

It's a physical properties just like the poison ration, young modulus, etc. It makes sense to be not equal to zero.